docs/arvo/basic.md

```diff
@@ -207,14 +217,14 @@ takes two children:
 As above, `++three` takes an integer argument, `a`, and then executes
 the remainder of the code with `a` set to the actual arguments.
 
-Similarly, `=|` ('[tisbar](/hoon/twig/tis-flow/bar-new/)') pushes its first child, `b` into our context
+Similarly, `=|` ('[tisbar](../../hoon/twig/tis-flow/bar-new/)') pushes its first child, `b` into our context
 (in other words, it declares a variable `b`) and executes the remainder
 of the code. However, `b` is not an argument; `=|` sets `b` to the
 default value of whatever type it is declared as. Since the default
 value of an atom is `0`, b is set to `0`.
 
-> To produce the default value of any given type, use `$*` ('buctar')
-> followed by the type. Alternatively, use the irregular form `*`.
+> To produce the default value of any given type, use `*` ('tar')
+> followed by the type. This operaction is called *bunt*.
 >
```

Comment:
As I understand it, `$*` is now (or always was?) `%$`, but that doesn't work either.  The irregular form is certainly `*`, so I agree with your change.